app/assets/stylesheets/scoreboards/_show.scss

```diff
@@ -144,3 +144,43 @@ $bottom-bar-height: 120px;
 
 .scoreboard-home-player-avatar { right: 0; }
 .scoreboard-away-player-avatar { left: 0;  }
+
+.scoreboard-warm-up {
+  @include padding(30px);
```

Comment:
If you're setting the padding for all of the sides you don't need to use the include. `padding: 30px;` would do the same thing.